addons/api/package.json

```diff
@@ -19,7 +19,7 @@
     "format": "yarn run format:js",
     "format:js": "prettier --write '{addon,app,config,stories,tests,mirage}/**/*.js' *.js",
     "start": "ember serve",
-    "test": "ember test",
+    "test": "ember exam --split=4 --parallel=1 --random",
```

Comment:
To keep `--random` and have it actually use a random seed, it has to be the last flag otherwise it will interpret anything after as a seed. 

Interestingly though, `--parallel` also takes in an argument so it wants to be last as well so I have to set the argument to `--parallel` as `1`. The confusing part is changing the value for `--parallel` doesn't actually do anything without using `--load-balance` since it respects the `parallel` value from testem, so `--parallel=1` doesn't actually do anything and is the same as just `--parallel`. See [this explanation](https://github.com/ember-cli/ember-exam/issues/270#issuecomment-489134727).

I chose to split it by 4 as it seems like [github runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) are on 2 core cpus? Not sure if it has 4 threads but I thought `4` was a reasonable number to start with to see how it runs in our CI. Let me know if you think it should be different, for context cloud ui uses 8.